auto_docking/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>auto_docking</name>
+  <version>0.0.0</version>
+  <description>Auto docking functionality for mobile robots</description>
+  <maintainer email="support@husarion.com">Husarion</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <depend>geometry_msgs</depend>
+  <depend>aruco_opencv_msgs</depend>
+  <depend>aruco_opencv</depend>
```

Comment:
This makes our package dependent on the type of the tag detected. We should consider using TF or generic Pose

Reply:
Yep, correct. We decided to go with a specific package for pose estimation so we use its API.

![Screenshot from 2023-09-12 08-55-21](https://github.com/husarion/auto_docking_ros/assets/108666440/dff8edac-6e49-42f8-84c6-a8237c52fcb4)

Relying on tf would lead us to use a timer to retrieve tf sequentially or subscribing the whole tf content available in our system (which is awkward)

Reply:
Ok ok